src/Controller/Api/RoutingController.php

```diff
@@ -280,6 +281,50 @@ public function activityTypes(): JsonResponse
         throw new ApiMethodDeniedException('The method used is not supported');
     }
 
+    /**
+     * Brokers
+     *
+     * @Route("/brokers", name="brokers")
+     */
+    public function brokers(): JsonResponse
```

Comment:
Please delete this method and set it up, as with the units. When an ID is passed, it is automatically a `readOne` call.

https://github.com/oveleon/contao-propstack-api-bundle/blob/137485b11c08237f2dd0bb5ec8bf7e6ed3d76b0e/src/Controller/Api/RoutingController.php#L109-L117